src/parser/statement.js

```diff
@@ -51,8 +51,12 @@ module.exports = {
         return this.read_trait();
       case this.tok.T_USE:
         return this.read_use_statement();
-      case this.tok.T_CONST:
-        return this.next().read_const_list();
+      case this.tok.T_CONST: {
+        const result = this.node("constantstatement");
+        const items = this.next().read_const_list();
+        this.expectEndOfStatement();
+        return result(items);
```

Comment:
here you need on the first parameter to pass null (related to the kind parameter)

Reply:
@ichiriac fixed already :+1: